node.js/week1/homework/src/backend/app.js

```diff
@@ -1,12 +1,81 @@
 const express = require("express");
 const app = express();
 
+
 // import data here
 const meals = require("./data/meals");
+const reviews = require("./data/reviews");
+const reservations = require("./data/reservations");
+
+//render the data in meals route 
+const  meals_reviewsJson = meals.map(meal => {
+  const mealsReview = reviews.filter(review => review.mealId === meal.id)
+  meal.reviews = mealsReview
+  return meal
+}) 
+
+//render the data in cheap_meals route
+const  cheapMeal = meals_reviewsJson.filter(meal => meal.price < 80)
+
+//render the data in large_meals route 
+const  largeMeal = meals_reviewsJson.filter(meal => meal.maxNumberOfGuests > 4 )
+
+//render the data of a random meal in meal route 
+function  randomNumber (obj) { // what can  name the obj ?
+  return Math.floor(Math.random() * obj.length) + 0;
```

Comment:
```suggestion
  return Math.floor(Math.random() * obj.length) ;
```
you can just write this expression without 0